lib/ziti.c

```diff
@@ -30,6 +30,14 @@ typedef struct ziti_dump_ctx_s {
     ziti_printer_cb_wrapper printer;
 } ziti_dump_ctx;
 
+ZITI_FUNC extern void
+ziti_logger(int level, const char *module, const char *file, unsigned int line, const char *func,
```

Comment:
I'm assuming there is a reason for not getting this from ziti_log.h.

Reply:
yes.  There's a conflict with `XX(DEBUG)`
![image](https://user-images.githubusercontent.com/19175177/165172209-15ca259c-232b-4a17-aeb5-629567126a4e.png)